parity-bytes/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Parity Technologies <admin@parity.io>"]
 repository = "https://github.com/paritytech/parity-common"
 description = "byte utilities for Parity"
 license = "GPL-3.0"
+edition = "2018"
```

Comment:
I guess there's not much else to do for this one huh?

Reply:
Right, unless you know how to avoid 
https://github.com/paritytech/parity-common/blob/73ffdab835c60be2b2d56d1112dafea200626868/parity-bytes/src/lib.rs#L25-L30 in 2018 edition.

Reply:
Exactly. I don't and I wonder if it'd even be more readable.